### DISCUSSION TITLE
Give us a way to Thank you!

### DISCUSSION THREAD

**Discussion by Josephkready** [manutalcual/winredock#48] [Show and tell]
This tool worked seamlessly, and probably saved me from spendings hundreds of dollars from buying a dedicated computer setup to ensure windows remain in the same location. Is there a Patrenon or coffee we can support you all with?

**Comment by VermilionBoulder** [comment]
I second this. It's amazing and Just Works™.
I have a hotdesk system at work so each docking station requires an initial setup, but whenever I come back to one I've already visited, all my windows just slide into place.
This is worth money :D

**Comment by royboyroyboy** [comment]
AMEN! This is the only thing I found that sounded remotely like it was built for purpose for this job, and also for me it just works, and works perfectly, exactly how I expected it to with no pomp and ceremony. Love love love.
